vicluster-core/src/main/java/org/gridkit/vicluster/telecontrol/Classpath.java

```diff
@@ -325,7 +330,49 @@ public long size() {
 		public String toString() {
 			return filename;
 		}
-	}	
+
+		public synchronized boolean isGridKitClasses() {

```

Comment:
Where should I move it? Probably it can be placed in ClasspathUtils? Can ClasspathEntry hold boolean fields isGridKitClasses and isTestClasses or such links should be moved to separate caches?